core/dispatcher/src/main/scala/whisk/core/container/ContainerUtils.scala

```diff
@@ -67,14 +67,15 @@ trait ContainerUtils extends Logging {
      */
     def makeContainer(name: Option[String], image: String, network: String, env: Map[String, String], args: Array[String], limits: ActionLimits, policy: Option[String])(implicit transid: TransactionId): ContainerId = {
         val nameOption = if (name isDefined) Array("--name", name.getOrElse("")) else Array.empty[String]
+        val cpuArg = Array("-c", s"${ContainerPool.cpuShare}")
```

Comment:
Why the string formatting? You could write just `ContainerPool.cpuShare.toString`